### PR TITLE
Added Stickler and Black configuration

### PR DESCRIPTION
**Pull request by SR4ven**
Preparations for #304 

TODO:
- [x] Port existing code to Black standard

Optimal features:
- [x] Automated commit with code style fixes
- [ ] Tox integration

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -5,7 +5,10 @@ zip_ok = False
 python-file-with-version = boofuzz/__init__.py
 
 [flake8]
-ignore = F403, F405, W503
+ignore = F403, F405, W503, E203
```
Comment (jtpereyda) [inline setup.cfg]: Why ignore E203?

**Comment by jtpereyda** [comment]
@SR4ven This is fantastic. I'll set up Stickler.

The auto-commit for fixes sounds really cool. Let's do it.

The tox integration makes me a bit more nervous as it could edit pre-commit files, but I think it's still worth a shot!

File: setup.cfg (hunk 1 above)
Comment (SR4ven) [reply setup.cfg]: See https://github.com/python/black#slices

**Comment by SR4ven** [comment]
@jtpereyda I see you already enabled Stickler, but it doesn't have a config file yet, so it fails the whole check.
We have two options now:
1. You merge this PR in the current state without any optimal features or updating existing code. I could take care of that in a seperate PR.
    I couldn't find any info about whether Stickler is checking only the diffs, whole touched files or the whole repo. Without updating the existing code this could possibly result in tests failing because a not modified file is not style conform.
2. I can definetly finish updating the existing code and enable the auto commit feature today.
    I need to think of a way to have Travis ignore Black in tox first before I can add that. I'll do this in another PR.

If getting current CI tests to pass is not that urgent, option 2 might be the safer one.
On the other hand it would be nice to know if Stickler is smart enough to only check the diff. We could easily find that out with option 1. Even if Stickler check all code everytime, it's not too much of a problem as the whole repo should follow the same style guide, which would be ensured that way.
Making the existing code Black conform means a lot of changes, but isn't that much work actually.

**Comment by SR4ven** [comment]
@jtpereyda if you agree with all style changes, we are ready to merge.

**Comment by jtpereyda** [comment]
@SR4ven Sorry for the long delay on this -- merging now!

**Pull request merged**